blueprints/lightning-deploy-config/files/config/deploy.js

```diff
@@ -15,14 +15,20 @@ module.exports = function(deployTarget) {
       prefix: '<%= dasherizedPackageName %>'
     }
   };
+
   if (VALID_DEPLOY_TARGETS.indexOf(deployTarget) === -1) {
     throw new Error('Invalid deployTarget ' + deployTarget);
   }
 
   if (deployTarget === 'dev') {
     ENV.build.environment = 'development';
     ENV.redis.url = process.env.REDIS_URL || 'redis://0.0.0.0:6379/';
-    ENV.plugins = ['build', 'redis']; // only care about deploying index.html into redis in dev
+    // only care about deploying index.html into redis in dev
+    ENV.pipeline = {
+      disabled: {
+        allExcept: ['redis']
```

Comment:
Not sure if we need `['build', 'redis']` here. In my project just having `['redis']` seems to work.

Reply:
Likely only because you've previous built to the `tmp/deploy-dist` directory.